docs/reference/modules/asset.md

```diff
@@ -36,7 +36,11 @@ When the `APOS_UPLOADFS_ASSETS` environment variable is present, this optional p
 
-Assets within the `ui/apos` folder modify the admin UI. Code to be passed to the build process should be organized into two subfolders, a `components` folder that contains any new Vue components, and a `apps` folder that takes any additional admin-facing JavaScript. During the build process code located in the `ui/apos` subdirectory of any module is automatically detected and incorporated. Assets in the `components` folder are registered automatically by name and do not need to be imported. Unlike assets in the `ui/src` folder where only the `index.js` file is an entry point, all files in the `ui/apos/apps` folder are entry points. If this behavior is undesirable any files that should not be entry points can be placed into a sub-directory and imported into the main entry point file. See the [`@apostrophecms/login` module](https://github.com/apostrophecms/apostrophe/tree/main/modules/%40apostrophecms/login) for an example, including import of the Vue library within the [`AposLogin.js` file](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/login/ui/apos/apps/AposLogin.js). Customizing the admin UI is covered in-depth in the [guide documentation](https://v3.docs.apostrophecms.org/guide/custom-ui.html#customizing-the-user-interface).
+Assets within the `ui/apos` folder modify the admin UI. Code to be passed to the build process should be organized into three subfolders, a `components` folder that contains any new Vue components, an `apps` folder that takes any additional admin-facing JavaScript, and a `tiptap-extensions` folder that contains any tiptap-extensions used within the admin UI.
+
+During the build process, code located in the `ui/apos` subdirectory of any module is automatically detected and incorporated. Assets in the `components` folder are registered automatically by name and do not need to be imported. Unlike assets in the `ui/src` folder where only the `index.js` file is an entry point, all files in the `ui/apos/apps` folder are entry points. If this behavior is undesirable any files that should not be entry points can be placed into a sub-directory and imported into the main entry point file. See the [`@apostrophecms/login` module](https://github.com/apostrophecms/apostrophe/tree/main/modules/%40apostrophecms/login) for an example, including import of the Vue library within the [`AposLogin.js` file](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/login/ui/apos/apps/AposLogin.js). Customizing the admin UI is covered in-depth in the [guide documentation](https://v3.docs.apostrophecms.org/guide/custom-ui.html#customizing-the-user-interface).
+
+The code in the `tiptap-extensions` folder is typically used to extend the `rich-text-widget`, which uses the [tiptap WYSIWYG editor](https://tiptap.dev/) at its core. This editor has a wealth of pre-made [extensions](https://tiptap.dev/extensions) that could be incorporated.
```

Comment:
Current language suggests it's loosey-goosey but it's more specific and only the right stuff will work, so:

"Every file in the `tiptap-extensions` folder must be a tiptap WYSIWYG rich text editor extension, written in that format. Every such extension will be loaded and made available for potential activation by the developer in the rich text widget's editing toolbar."